### PR TITLE
Remove Rivtech 20x66 Ammo and Guns 

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Bugfixes "Remove Rivtech 20x66 Ammo and Guns "

#### Purpose of change
Caseless shotgun ammo would be exorbitantly difficult to engineer, manufacture, and market to the point it wouldn't realistically exist in any amounts in CDDA. They're a holdover of an older part of CDDA, and doesn't fit with the modern game. Below is a list of the reasons why I've removed 20x66 ammo. Note that this doesn't really touch on issues with the guns mechanically, just the ammo they use.

Why it’s bad, mechanically

-Main benefit of caseless ammo is weight reduction, shotgun ammo is carried in low amounts so benefit isn’t great
-Overpowered, it’s roughly equivalent to 10 gauge ballistically and so all the weight reductions from caseless is undone by it being such a magnum of a round 
-Buckshot accuracy would likely be lowered. Shotgun cases hold in the buckshot with a crimp, something caseless ammo couldn’t do as it can’t be crimped. Some sort of adhesive could be used, but that’d impact performance and spread as it’d stick to some shot, making them slow down faster and diverge due to being aerodynamically unstable
-Militaries don’t just use the most powerful round possible. If the US Army so wished, they could be using 3 ½ shells instead of 2 ¾ but don’t, because the extra power is just unneeded
-Buckshot relies on a shotcup to help increase accuracy, and being caseless it’d be much more difficult to engineer a way for a shotcup to exist and work

Why it’s bad, marketwise

–10 gauge is specifically out of favour because it’s quite expensive, requires a new gun to shoot it, and isn’t used much as the high power isn’t all that useful for the majority of shotgun uses. 3 ½ 12 gauge shells are much more favourable and are ballistically similar to 10 gauge. Rivtech has designed what is a 10 gauge line of ammo that is way more expensive, and requires an even more expensive gun
-The main purpose of magnum rounds like 3 ½ or 10 gauge is for turkey hunting. The market demand for magnum shotgun rounds for non-hunting purposes is quite small, commercially and militarily so Rivtech would be pouring in a lot of money into guns that nobody is going to use seriously
-Being caseless makes it impossible to reload, so less people are going to get into using that calibre
-Being caseless makes it way more expensive than usual shotgun rounds, due to it being a new type of powder, requiring very specific factory lines, and can’t use much existing tooling to make them
-People who would use a lot of ammo such as skeet shooters or hunters likely wouldn’t trust caseless shotgun ammo
-Due to 1 company manufacturing ammo, availability for it would be very poor
-Rivtech is trying to supplant themselves in a market who’s calibres has existing for decades, would be a very uphill battle to get people to use it 
-As these are likely rimless, tolerances for manufacturing have to be a lot tighter due to headspacing, further driving up the cost of manufacturing

TL;DR
-Very hard to justify using 20x66 instead of 3 ½ or 10 gauges. The ammo is rarer, more expensive, less guns have them
-Militaries don’t use the most powerful round possible 
-The design of the round doesn’t make much sense
-Would have a tough time in the market

#### Describe the solution

Removes them.

#### Describe alternatives you've considered

I didn't. Anything that could be better than this would still require removal of the 20x66 anyways to properly replace it. 

#### Testing
Hub 01 Security Head Weapon
image
Before and after: (they still exist and work)
image
image

#### Additional context

To any merger, I'm waiting on Kevin's review. Just to be safe as I'm tackling a very old and ingrained part of CDDA.

### CODE REVIEW

**Comment by DukePaulAtreid3s** [comment]
What's the difference between the two pictures?

**Comment by Holli-Git** [comment]
> What's the difference between the two pictures?

Nothing. That's the point, to show that they've been properly moved to the obsoletion folder instead of deleted.

**Comment by TheShadowFerret** [comment]
>Describe alternatives you've considered
>I didn't

You could migrate all of them into 12ga ammo/guns/mags 🤔

**Comment by Holli-Git** [comment]
> You could migrate all of them into 12ga ammo/guns/mags so players w/ existing games can combine their migrated supplies w/ extant ones 🤔
> 
> (Not a big deal though if you don't)

Can't, there's not a big enough of a magazine for the magazines. Also the underbarrel shotgun runs into issues as well

**Comment by PatrikLundell** [comment]
I don't really care if this is removed, but some devil's advocacy for you:
- Argument that shotguns don't use enough ammo for weight to be an issue, and then claim that it's bad that the weight gain is eaten up by an increased caliber. You'd get more (over?) power for the same weight.
- My understanding of RivTech was that it's an engineering geek company where technical finesse is more important than usability. In addition to that, these geeks had deep ties to military procurement, allowing for sale of expensive and impractical weapons to very deep pockets on a long term contract. In that context, civilian use would be unimportant. It wouldn't be the first time military customers have bought equipment in that fashion only to find that it's impractical and doesn't really work reliably in practice (and at times being very disliked by those who where issued the equipment).

**Comment by NerosShadow** [comment]
> I don't really care if this is removed, but some devil's advocacy for you:
> 
>     * Argument that shotguns don't use enough ammo for weight to be an issue, and then claim that it's bad that the weight gain is eaten up by an increased caliber. You'd get more (over?) power for the same weight.
> 
>     * My understanding of RivTech was that it's an engineering geek company where technical finesse is more important than usability. In addition to that, these geeks had deep ties to military procurement, allowing for sale of expensive and impractical weapons to very deep pockets on a long term contract. In that context, civilian use would be unimportant. It wouldn't be the first time military customers have bought equipment in that fashion only to find that it's impractical and doesn't really work reliably in practice (and at times being very disliked by those who where issued the equipment).

I think that taking the Military Industrial Complex into account justifies this as well. Not to mention that every point made against the Rivtech ammo and firearms could be said for the XM7. Its heavy, expensive, and uses arguably overpowered ammunition to achieve ballistic capabilities slightly over existing weapon systems. 
That being said, we need to add the XM7 and XM250 ASAP.

**Comment by GuardianDll** [comment]
> we need to add the XM7 and XM250 ASAP

No we don't, there is still not as much of them to be represented. M4 needed ten years and entire gulf war to establish itself and substantially replace M16 
And XM7 was adopted only two years ago...

**Comment by Holli-Git** [comment]
> I don't really care if this is removed, but some devil's advocacy for you:

A) That first part still runs head first into the issue of nobody really needs that power. If it weighs the same or more than 2 3/4 it still won't be used 

B) There's a huge leap from corruption to downright silliness. Ceramic superalloy guns? Caseless shotgun ammo? 40 round shotgun magazines? That's a poor argument that only degrades Rivtech, as it cements itself as a nonsense company who wouldn't actually exist. Like yeah, corruption does happen, but as bad as Rivtech? If there's a case of corruption on the level of Rivtech guns I'll eat my shorts, but I'm confident there's not.

**Comment by Holli-Git** [comment]
> I think that taking the Military Industrial Complex into account justifies this as well. Not to mention that every point made against the Rivtech ammo and firearms could be said for the XM7. Its heavy, expensive, and uses arguably overpowered ammunition to achieve ballistic capabilities slightly over existing weapon systems. That being said, we need to add the XM7 and XM250 ASAP.

A) Not really. Military industrial complex weapons tend to work well

B) That is a horrible argument that ignores both my arguments and the reasons the XM7/250 are a thing. 
-First of all, 6.8 is a rifle cartridge, 20x66 is a shotgun cartridge. Rifles are used in much different applications than shotguns, you don't need magnum shotgun rounds for doors, you do need powerful rifles to penetrate body armor and barriers.
-A huge point of the NGSW was to have a more powerful round to defeat body armor. It has an actual goal in mind and a method to achieve it, not Rivtechs pointless development. 
-Battle rifles exist. Caseless magazine fed automatic combat shotguns don't.

**Review by GuardianDll — APPROVED** [review]

**Comment by kevingranade** [comment]
Just a note for discussion,  we don't need a devils advocate. If you don't really believe that there's something wrong with a proposed change but disagree with some premise or argument about it,  just let it go.

**Comment by KingSaje** [comment]
@PatrikLundell thank you for at least trying to see the other side. It's heartbreaking sometimes to see the fun and interesting parts of CDDA that favour the player being ripped away in favour of realism. I'm not the only one who feels this way and I think it would go a long way in servicing the community to find a creative way to rework these "Old CDDA" holdovers instead of downright removing them.

**Comment by GuardianDll** [comment]
There was no way to rework it, and we already got a decent replacement of it in form of exodii guns, that do not rely on "being old content" as their only reason to stay in the game
For any future discussions please make a separate issue instead of writing in already merged PRs
